lms/templates/vert_module.html

```diff
@@ -1,4 +1,8 @@
 <%page expression_filter="h"/>
+<%! from openedx.core.djangolib.markup import HTML %>
+%if unit_title:
+  <h3 class="unit-title">${unit_title}</h3>
```

Comment:
@FYI to reviewers on this PR-- adding this header level impacted all existing headers on the page, as there are now two h3s. Inserting or removing header levels impacts child content, so the implications should be thought through.

See TNL-6039.